engines/scumm/he/net/net_lobby.cpp

```diff
@@ -429,7 +429,8 @@ void Lobby::login(const char *userName, const char *password) {
 	loginRequestParameters.setVal("user", new Common::JSONValue(_userName));
 	loginRequestParameters.setVal("pass", new Common::JSONValue((Common::String)password));
 	loginRequestParameters.setVal("game", new Common::JSONValue((Common::String)_gameName));
-	loginRequestParameters.setVal("version", new Common::JSONValue(gScummVMVersionLite));
+	loginRequestParameters.setVal("version", new Common::JSONValue(gScummVMFullVersion));
```

Comment:
I feel like `gScummVMVersionLite` should be removed from `base/version.cpp`, since it will not serve a purpose anymore.

Reply:
Agreed, I hadn't realized you'd added it just for that purpose. Done.